backend/custom_models/models.py

```diff
@@ -0,0 +1,8 @@
+from .language import Language
+from .item_type import ItemType
+from .item import Item
+from .item_text import ItemText
+from .question_type import QuestionType
+from .question import Question
+
+# These imports help to audo discover the models

```

Comment:
Little typo here: _audo_ instead of _auto_

Reply:
Fixed in dd37aae